data_management/models.py

```diff
@@ -645,26 +648,44 @@ class ExternalObject(BaseModel):
     `last_updated`: Datetime that this record was last updated
 
     `updated_by`: Reference to the user that updated this record
+
+    `version`: Version identifier of the `DataProduct` associated with this `ExternalObject`
     """
-    ADMIN_LIST_FIELDS = ('doi_or_unique_name', 'title')
+    ADMIN_LIST_FIELDS = ('other_unique_name', 'title', 'version')
```

Comment:
Not sure what this does, but should it include `identifier`?

Reply:
Fixed in https://github.com/FAIRDataPipeline/data-registry/commit/260cdb9af251da847c6995b5eaf8dd0a420a0f17